packages/tx/src/baseTransaction.ts

```diff
@@ -59,9 +75,40 @@ export abstract class BaseTransaction<TransactionObject> {
   }
 
   /**
-   * Checks if the transaction has the minimum amount of gas required
-   * (DataFee + TxFee + Creation Fee).
+   * Returns the transaction type
    */
+  get transactionType(): number {
+    return this._type
+  }
+
+  /**
+   * Alias for `transactionType`
+   */
+  get type() {
+    return this.transactionType
+  }
+
+  /**
+   * EIP-2930 alias for `r`
+   */
+  get senderR() {
+    return this.r
+  }
+
+  /**
+   * EIP-2930 alias for `s`
+   */
+  get senderS() {
+    return this.s
+  }
+
+  /**
+   * EIP-2930 alias for `v`
+   */
+  get yParity() {
```

Comment:
I don't think this makes sense here. yParity is `1` or `0`, and `v` can have other values in the legacy txs.

Reply:
@ryanio if you have time it would be great if you move over again (then all three alias methods for consistency) to the `EIP2930`tx class in the release PR.

Reply:
@holgerd77 sure, done in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1192/commits/6078234526d79721e7bc9b41e0faa7fa1bbe9dfb

Reply:
thanks, sorry for the back-and-forth